src/java/org/apache/cassandra/tools/nodetool/Info.java

```diff
@@ -51,6 +51,8 @@ public void execute(NodeProbe probe)
         out.printf("%-23s: %s%n", "Gossip active", gossipInitialized);
         out.printf("%-23s: %s%n", "Native Transport active", probe.isNativeTransportRunning());
         out.printf("%-23s: %s%n", "Load", probe.getLoadString());
+        out.printf("%-23s: %s%n", "Uncompressed load", probe.getUncompressedLoadString());
```

Comment:
Given we're adding this at the end of the existing output, and we're doing this in trunk/a major release, this seems fine.